docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -0,0 +1,344 @@
+
+-- In FULL mode, the interval is not a common divisor of the respective time range
+FRESHNESS = INTERVAL '60' SECOND
+FRESHNESS = INTERVAL '5' HOUR
+```
+
+<span class="label label-danger">Note</span>
+- The materialized table data will be refreshed as closely as possible within the defined freshness but cannot guarantee complete satisfaction.
+- In FULL mode, data freshness must be translated into a cron expression, consequently, only freshness intervals within predefined time spans are presently accommodated, this design ensures alignment with cron's capabilities. Specifically, support for the following freshness:
+    - Second: 30, 15, 10, 5, 2, and 1 second intervals.
+    - Minute: 30, 15, 10, 5, 2, and 1 minute intervals.
+    - Hour: 8, 4, 2, and 1 hour intervals.
+    - Day: 1 day.
+
+### REFRESH_MODE
+
+REFRESH_MODE is used to explicitly specify the refresh mode of the materialized table. The specified mode takes precedence over the framework's automatic inference to meet specific scenarios' needs.
+
+**Examples:**
+(Assuming `materialized-table.refresh-mode.freshness-threshold` is 30 minutes)
+
+```sql
+-- The refresh mode of the created materialized table is CONTINUOUS, and the job's checkpoint interval is 1 hour.
+CREATE MATERIALIZED TABLE my_materialized_table
+    REFRESH_MODE = CONTINUOUS
+    FRESHNESS = INTERVAL '1' HOUR
+    AS SELECT
+       ...    
+
+-- The refresh mode of the created materialized table is FULL, and the job's schedule cycle is 10 minutes.
+CREATE MATERIALIZED TABLE my_materialized_table
+    REFRESH_MODE = FULL
+    FRESHNESS = INTERVAL '10' MINUTE
+    AS SELECT
+       ...    
+```
+
+### AS <select_statement>
+
+This clause used to fill the materialized table with data from the select query. The upstream table can be a materialized table, table, or view. The select statement supports all Flink SQL [Queries]({{< ref "docs/dev/table/sql/queries/overview" >}}).
+
+**Example:**
+
+```sql
+CREATE MATERIALIZED TABLE my_materialized_table
+    FRESHNESS = INTERVAL '10' SECOND
+    AS SELECT * FROM kafka_catalog.db1.kafka_table;
+```
+
+## Examples
+
+Create a materialized table with a data freshness of 10 seconds and a refresh mode of CONTINUOUS:
+
+```sql
+CREATE MATERIALIZED TABLE my_materialized_table_continuous
+    PARTITIONED BY (ds)
+    WITH (
+        'format' = 'json',
+        'partition.fields.ds.date-formatter' = 'yyyy-MM-dd'
+    )
+    FRESHNESS = INTERVAL '10' SECOND
+    REFRESH_MODE = 'CONTINUOUS'
+    AS 
+    SELECT 
+        k.ds,
+        k.user_id,
+        COUNT(*) AS event_count,
+        SUM(k.amount) AS total_amount,
+        MAX(u.age) AS max_age
+    FROM 
+        kafka_catalog.db1.kafka_table k
+    JOIN 
+        user_catalog.db1.user_table u
+    ON 
+        k.user_id = u.user_id
+    WHERE 
+        k.event_type = 'purchase'
+    GROUP BY 
+        k.ds, k.user_id
+```
+
+Create a materialized table with a data freshness of 10 minutes and a refresh mode of FULL:
```

Comment:
Create a materialized table with a data freshness of 1 hour and the derived refresh mode is FULL: